### PR TITLE
GrypeDBMetadata Persistence Model Changes

### PR DESCRIPTION
**Pull request by Vijay-P**
Removes foreign key constraints on GrypeDBMetadata and changes name of "checksum" column to "archive_checksum".

### CODE REVIEW

File: anchore_engine/services/policy_engine/engine/feeds/grypedb_sync.py [hunk 1]
```diff
@@ -136,20 +136,22 @@ def _update_grypedb(
         try:
             if grypedb_file_path:
                 GrypeWrapperSingleton.get_instance().init_grype_db_engine(
-                    grypedb_file_path, active_grypedb.checksum
+                    grypedb_file_path, active_grypedb.archive_checksum
```
Comment (dspalmer99) [inline anchore_engine/services/policy_engine/engine/feeds/grypedb_sync.py:139]: This call will change slightlyt post https://github.com/anchore/anchore-engine/pull/1048 to include the db schema version

File: anchore_engine/services/policy_engine/engine/feeds/grypedb_sync.py [hunk 2]
```diff
@@ -136,20 +136,22 @@ def _update_grypedb(
...
+                GrypeDBFile.verify_integrity(
+                    grypedb_document, active_grypedb.archive_checksum
+                )
                 with GrypeDBStorage() as grypedb_file:
-                    with grypedb_file.create_file(active_grypedb.checksum) as f:
+                    with grypedb_file.create_file(active_grypedb.archive_checksum) as f:
                         f.write(grypedb_document)
                     GrypeWrapperSingleton.get_instance().init_grype_db_engine(
-                        grypedb_file.path, active_grypedb.checksum
+                        grypedb_file.path, active_grypedb.archive_checksum
```
Comment (dspalmer99) [inline anchore_engine/services/policy_engine/engine/feeds/grypedb_sync.py:154]: Same here

File: feed_sync_updates__Signed-off-by__Vijay_Pillai__vijay_pillai@anchore_com_.patch [hunk 3]
```diff
@@ -0,0 +1,273 @@
+Index: tests/integration/services/policy_engine/test_vulnerabilities.py
```
Comment (dspalmer99) [inline feed_sync_updates__Signed-off-by__Vijay_Pillai__vijay_pillai@anchore_com_.patch:1]: Did this get committed accidentally?

File: anchore_engine/services/policy_engine/engine/feeds/grypedb_sync.py (hunk 1 above)
Comment (Vijay-P) [reply anchore_engine/services/policy_engine/engine/feeds/grypedb_sync.py:139]: will rebase after 1048 merge

**Pull request closed without merging**